  File "/Users/sogood/Downloads/dearfile/a.py", line 1163, in operation
    line.sendMessage(to, 'TalkException Error, Code: ' + str(e.code) + '\nReason: '+ str(e.reason))
  File "/Users/sogood/Downloads/dearfile/lib/linepy/talk.py", line 17, in checkLogin
    return func(*args, **kwargs)
  File "/Users/sogood/Downloads/dearfile/lib/linepy/talk.py", line 120, in sendMessage
    return self.talk.sendMessage(self._messageReq[to], msg)
  File "/Users/sogood/Downloads/dearfile/lib/akad/TalkService.py", line 9293, in sendMessage
    return self.recv_sendMessage()
  File "/Users/sogood/Downloads/dearfile/lib/akad/TalkService.py", line 9318, in recv_sendMessage
    raise result.e
